com.protonvpn.ProtonVPN.yml

```diff
@@ -14,6 +14,8 @@ finish-args:
   - --share=network
   - --socket=wayland
   - --socket=fallback-x11
+  # https://github.com/ProtonVPN/linux-app/blob/23350af7d5117784141786ef88fa58c47a9dbfdc/protonvpn_gui/main.py#L39
+  - --own-name=com.protonvpn.www
```

Comment:
Please just name the json file and everything to the correct id.

Reply:
I do, but it seems the app id was choosen incorrectly by upstreams. It might be a good idea to point out that it should be com.protonvpn.protonvpn or com.protonvpn.ProtonVPN or similar.